core/services/relay/relay.go

```diff
@@ -0,0 +1,33 @@
+package relay
+
+import (
+	uuid "github.com/satori/go.uuid"
+	"github.com/smartcontractkit/chainlink/core/service"
+	"github.com/smartcontractkit/libocr/offchainreporting2/reportingplugin/median"
+	"github.com/smartcontractkit/libocr/offchainreporting2/types"
+)
+
+type Network string
+
+var (
+	EVM               Network = "evm"
+	Solana            Network = "solana"
+	SupportedRelayers         = map[Network]struct{}{
```

Comment:
Duplicate source of truth for available relays, and should probably not mix with interfaces here.

Also, a rely can exist for Solana, but it only has implemented OCR2 & CCIP providers, no Keeper. So if a Keeper job uses network relayers to run (let's imagine it does), then the job spec validation should fail for Keeper Solana jobs, but pass for others that are supported by the relay.

I'd suggest this information is available from the master Relayer delegate, where all available relays are implemented. So basically to validate any job spec we'd need to fetch info from the master Relayer delegate.

Perfectly ok to leave it like this for this initial situation of 1x job + 2x relay, but consider adding a `// TODO` for the next iteration.

Reply:
Yeah agreed I don't like them here either, unfortunately its tricky to have them in the master relay delegate because of various import cycles. I'll see what I can do about those

Reply:
We're aways away from making these relays general purpose, so I'd hold off any TODOs until there's a clear roadmap for that